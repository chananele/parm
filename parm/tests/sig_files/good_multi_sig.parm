---
name: good_multi_p1
exports:
  - after_main
method: match
pattern: |
  % goto(0x103E8)
  ...
  BX LR
  after_main:
---
name: good_multi_p2
imports:
  - after_main
exports:
  - init_proc
method: match
pattern: |
  % goto("after_main")
  ... {2, 10}
  BL @:init_proc
  ... {2}
  SUB *
...
