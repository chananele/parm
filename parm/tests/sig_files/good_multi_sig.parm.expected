name: good_multi_p1
result: pass
matches:
  after_main: 66564
---
name: good_multi_p2
result: pass
matches:
  init_proc: 66172
